packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -116,6 +116,16 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 }
 
+// Move label when textfield gets autofilled in Chrome
+.mdc-textfield__input {
+  // stylelint-disable plugin/selector-bem-pattern
+  &:-webkit-autofill + .mdc-textfield__label {
```

Comment:
Nope, it doesn't prefix autofill pseudos. You can test in [an online playground](https://autoprefixer.github.io/).